src/celeritas/ext/detail/GeantVolumeVisitor.cc

```diff
@@ -30,7 +30,7 @@ namespace detail
  * duplicated.
  * Function called by \c store_volumes(...) .
  */
-void GeantVolumeVisitor::visit(G4LogicalVolume const& logical_volume)
+void GeantVolumeVisitor::visit(G4LogicalVolume& logical_volume)
```

Comment:
Since we're not actually modifying the LV—the code we're calling is simply not const correct—leave the interfaces as const and then `const_cast<LV*>(&lv)` at the last possible moment.